exercises/reverse-string/runtests.jl

```diff
@@ -0,0 +1,20 @@
+# canonical data version: 1.2.0
+using Test
+
+include(reverse-string.jl)
```

Comment:
```suggestion
include("reverse-string.jl")
```

You forgot quotes here.

Please make sure to run the tests locally before submitting a PR, this should've been caught by that